README.md

```diff
@@ -0,0 +1 @@
+[![Build Status](https://dev.azure.com/event-maker-telegram-bot/Core/_apis/build/status/Validate%20the%20code?branchName=master)](https://dev.azure.com/event-maker-telegram-bot/Core/_build/latest?definitionId=2&branchName=master)
```

Comment:
@fidelman I am afraid it won't be correctly displayed, since there won't be any build on the `master` branch. The builds will happen on `feature/*` branch when there is a PR to merge it into `develop`, and on `develop` branch, when merging it into `master`. To fix this, please check other options, e.g. additional CI run for the `master` branch (`branch` section in `.yml` config), or maybe the status will be updated during `master` branch deployment (in that case no additional changes required).

Reply:
> @fidelman I am afraid it won't be correctly displayed, since there won't be any build on the `master` branch. The builds will happen on `feature/*` branch when there is a PR to merge it into `develop`, and on `develop` branch, when merging it into `master`. To fix this, please check other options, e.g. additional CI run for the `master` branch (`branch` section in `.yml` config), or maybe the status will be updated during `master` branch deployment (in that case no additional changes required).

I see, so let's start from when we even need that badge.
We need it only to show on the main page of the repo to represent _something_. We do not need it for development reasons or something like that.
For example, we can use it for:
- We create PR into master, this pipeline is failed but we merged it anyway somehow. So we will see, aha our master includes invalid code.
Honestly, otherwise, I do not see any reason to have it, especially for this pipeline. It will be very useful for deployment build, but we do not have it now.
What about to remove this badge, bcz it does not bring so many information?

Reply:
Yes, let's remove it, that's quicker. For deployment there should be another build on `master`, which could be the source of this information.